ui/src/shared/utils/useVisDomainSettings.ts

```diff
@@ -31,15 +38,14 @@ export const getValidRange = (
 export const useVisDomainSettings = (
   storedDomain: number[],
   data: NumericColumnData,
-  startTime: number = Infinity,
-  endTime: number = -Infinity
+  timeRange: TimeRange | null = null
```

Comment:
since we are using the `useVisDomainSettings` function to determine the `y` axis and aren't passing the timerange in those cases, we need to set a default

Reply:
ok